RunCat365/Program.cs

```diff
@@ -158,6 +146,20 @@ public RunCat365ApplicationContext()
             current = 1;
         }
 
+        private ToolStripMenuItem ChargeToolStripMenu<T>(
+            string title,
+            Func<T, string> obtainText,
+            EventHandler onClickEvent,
+            Func<T, bool> isChecked ) where T : Enum
+        {
+            var items = new List<ToolStripMenuItem>();
+            foreach (T value in Enum.GetValues(typeof(T)))
+            {
+                items.Add(new ToolStripMenuItem(obtainText(value), null, onClickEvent) { Checked = isChecked(value) });
```

Comment:
Please respect the original format.

```suggestion
                var item = new ToolStripMenuItem(obtainText(value), null, onClickEvent) 
                { 
                    Checked = isChecked(value)
                };
                items.Add(item);
```